task/sql-tasks.js

```diff
@@ -177,7 +274,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            COUNT(*) AS 'TotalOfCurrentProducts',
+            SUM(Discontinued) AS 'TotalOfDiscontinuedProducts'

```

Comment:
use count to show intent explicitly

in this case it's better to just use 2 select subqueries

from-clause is optional.

 it's better to write this query like this

```
select
   (select count()...) as 'T'
   (select count()...) as 'TT'
```